clang-3.4 and earlier has a bug with intitialization of QStringList with initialization lists
--- src/daemon/Authenticator.cpp
+++ src/daemon/Authenticator.cpp
@@ -314,7 +314,7 @@
         process->setProcessEnvironment(env);
 
         // start session
-        process->start(Configuration::instance()->sessionCommand(), { command });
+        process->start(Configuration::instance()->sessionCommand(), QStringList(command));
 
         // connect signal
         connect(process, SIGNAL(finished(int,QProcess::ExitStatus)), this, SLOT(finished()));

--- src/daemon/DisplayServer.cpp
+++ src/daemon/DisplayServer.cpp
@@ -60,6 +60,7 @@
         // log message
         qDebug() << " DAEMON: Display server starting...";
 
+        QStringList start_parameters;
 #if !TEST
         // set process environment
         QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
@@ -72,9 +73,11 @@
         Display *display = qobject_cast<Display *>(parent());
 
         // start display server
-        process->start(Configuration::instance()->serverPath(), { m_display, "-auth", m_authPath, "-nolisten", "tcp", QString("vt%1").arg(QString::number(display->vtNumber()), 2, '0')});
+        start_parameters << m_display << "-auth" << m_authPath << "-nolisten" << "tcp" << QString("vt%1").arg(QString::number(display->vtNumber()), 2, '0');
+        process->start(Configuration::instance()->serverPath(), start_parameters);
 #else
-        process->start("/usr/bin/Xephyr", { m_display, "-ac", "-br", "-noreset", "-screen",  "800x600"});
+        start_parameters << m_display << "-ac" << "-br" << "-noreset" << "-screen" << "800x600";
+        process->start("/usr/bin/Xephyr", start_parameters);
 #endif
         // wait for display server to start
         if (!process->waitForStarted()) {

--- src/daemon/Greeter.cpp
+++ src/daemon/Greeter.cpp
@@ -70,12 +70,15 @@
         env.insert("XCURSOR_THEME", Configuration::instance()->cursorTheme());
         process->setProcessEnvironment(env);
 
+        QStringList start_parameters;
 #if !TEST
         // start greeter
-        process->start(QString("%1/sddm-greeter").arg(BIN_INSTALL_DIR), { "--socket", m_socket, "--theme", m_theme });
+        start_parameters << "--socket" << m_socket << "--theme" << m_theme;
+        process->start(QString("%1/sddm-greeter").arg(BIN_INSTALL_DIR), start_parameters);
 #else
         // start greeter
-        process->start(QString("%1/sddm-greeter").arg("."), { "--socket", m_socket, "--theme", m_theme });
+        start_parameters << "--socket" << m_socket << "--theme" << m_theme;
+        process->start(QString("%1/sddm-greeter").arg("."), start_parameters);
 #endif
         // wait for greeter to start
         if (!process->waitForStarted()) {
